src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.logo {
```

Comment:
if the logo is BEM block then create the file logo.scss and move styles

Reply:
Fixed (remove BEM block for logo)